### DISCUSSION TITLE
[Enhance] Terrible UX when using touchpad

### DISCUSSION THREAD

**Discussion by mujiannan** [TabularEditor/TabularEditor3#1117] [Ideas]
**Current:**   
Working with TE3, if you use a touchpad, and the result of a query is quite wide, then the UX will be very terrible.  
| You want to watch content | You need to scroll |
| --- | --- |
| Up | Down |
| Down | Up |
| Left | Left |
| Right | Right |

This's quite terrible.

---

**Better:**  
| You want to watch content | You need to scroll |
| --- | --- |
| Up | Down |
| Down | Up |
| Left | Right |
| Right | Left |

**Comment by otykier** [comment]
Thanks for reporting. We have confirmed this issue and will provide an option in the next release, to allow reversing side-scroll.

**Comment by zenzeinet** [comment]
I reported the same with side-scrolling with the mouse, as it moves in the opposite direction as the rest of the Microsoft tools. Could you separate sideway scroll and up-down scroll?
